packages/api-gateway/src/drivers/http/adapters/places/handlers.js

```diff
@@ -1,17 +1,17 @@
 async function postPlace(req, reply) {
   const {
-    place_name, price_per_night_usd, host_id, type,
+    placeName, pricePerNightUsd, hostId, type,
   } = req.body;
 
   req.log.info('[http-server]: posting a place');
 
   await this.placesService.postPlace({
-    place_name, price_per_night_usd, host_id, type,
+    placeName, pricePerNightUsd, hostId, type,
```

Comment:
Based on https://github.com/Platzi-Master-C9/booking-services/pull/157/files#r850806915
```suggestion
    place_name: placeName,
    price_per_night_usd: pricePerNightUsd,
    host_id: hostId,
    type,
```